features/identity-event/org.wso2.carbon.identity.event.server.feature/resources/identity-event.properties

```diff
@@ -16,7 +16,7 @@
 # under the License.
 #
 
-threadPool.size = 10
+threadPool.size = 20
```

Comment:
Any specific reason for updating this value?

Reply:
The required event handlers were limited to the number thats given when binding. So it was increased to bind all the needed event handlers